app/src/main/java/org/permanent/permanent/ui/public/PublicGalleryFragment.kt

```diff
@@ -24,11 +29,17 @@ import org.permanent.permanent.viewmodels.PublicGalleryViewModel
 class PublicGalleryFragment : PermanentBaseFragment(), PublicArchiveListener {
     private lateinit var viewModel: PublicGalleryViewModel
     private lateinit var binding: FragmentPublicGalleryBinding
-    private lateinit var publicArchiveAdapter: PublicArchiveAdapter
+    private lateinit var yourArchivesAdapter: PublicArchiveAdapter
+    private lateinit var popularArchivesAdapter: PublicArchiveAdapter
     private lateinit var yourArchivesRecyclerView: RecyclerView
+    private lateinit var popularArchivesRecyclerView: RecyclerView
```

Comment:
instead of having 2 recyclerviews it would be better to have just one with one adapter with item types